docker/Dockerfile

```diff
@@ -176,13 +176,13 @@ ARG CUDA_MINOR_VER
 ARG CONDA_CHANNEL_ALIAS
 
 # Copy the development dependencies file
-COPY ${MORPHEUS_ROOT_HOST}/docker/conda/environments/cuda${CUDA_MAJOR_VER}.${CUDA_MINOR_VER}_dev.yml ./docker/conda/environments/
+COPY ${MORPHEUS_ROOT_HOST}/docker/conda/environments/dev_cuda-${CUDA_MAJOR_VER}${CUDA_MINOR_VER}_arch-x86_64.yml ./docker/conda/environments/dev.yaml
```

Comment:
Shouldnt the environments under `docker/conda` be deleted by now and we should be using the ones in `conda/environments`?

Reply:
That's correct. They have been deleted, which was the primary reason this was failing to build.